google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/sync/datacatalog_synchronizer.py

```diff
@@ -25,11 +25,10 @@
     import metrics_processor
 
 from google.datacatalog_connectors.rdbms import prepare
+from google.datacatalog_connectors.rdbms.common import constants
 from google.datacatalog_connectors.rdbms.prepare import\
```

Comment:
```suggestion
from google.datacatalog_connectors.rdbms.prepare import sql_objects
```
Can we simplify this?

Reply:
can you elaborate?